apps/fluent-tester/src/TestComponents/Button/ButtonVariantTestSection.mobile.tsx

```diff
@@ -32,11 +32,11 @@ export const ButtonVariantTest: React.FunctionComponent = () => {
       <Button appearance="subtle" disabled style={commonTestStyles.vmargin}>
         Subtle Disabled
       </Button>
-      <FAB icon={iconProps} iconOnly disabled style={commonTestStyles.vmargin} />
+      <FAB icon={iconProps} iconOnly disabled style={commonTestStyles.vmargin} accessibilityLabel="FAB" />
```

Comment:
is this related to a11y issue or just E2E fix ? 
I think we should add it in component not only on tester app, can you give bit context related to this ?

Reply:
A11y issue, this isn't the issue with other components on the page since they either have a text and that is picked up as the label